docs/source/ivr/ivr.rst

```diff
@@ -316,3 +316,11 @@ Custom exception
         }
 
     This will allow you to return an error code that your IVR provider can understand.
+
+Using IVR module with IVR providers
+===================================
+
+IVR module supports multiple IVR providers. For more information on how to integrate them with the module, please visit
+the links below.
+
+- :std:ref:`Using IVR with Verboice <using-ivr-with-verboice>`
```

Comment:
This doesn't generate a working link. I've given you a ready piece of code to paste that would generate a link.